containers/nginx-storage/install-dependencies.sh.j2

```diff
@@ -15,7 +15,9 @@ opm get spacewander/luafilesystem
 # Create .so symlink for libcrypto.so manually.
 # In normal circumstances the link would be created by ldconfig,
 # but in Alpine Linux ldconfig is non-functional.
-ln -s /usr/lib/libcrypto.so.41 /usr/lib/libcrypto.so
+# WARNING !!! If you update openresty version, remember to check
+# that `libcrypto.so.XX` wasn't changed.
+ln -s /usr/lib/libcrypto.so.42 /usr/lib/libcrypto.so
```

Comment:
Wouldn't it be better to say something like `"if you get error "<error message here>", you need to update the version number in libcrypto.so.XX"`? As it is now it seems to imply that you should not update it.

Reply:
Done